src/main/java/com/byndyusoft/template/client/configuration/ClientProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.byndyusoft.template.client.configuration;
+
+import java.time.Duration;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "clients.templateClient")
```

Comment:
I think that looks better: `clients.service-name`

Because usually don't write CamelCase in yml. We also make it clear that in our understanding, it is this configuration format that looks good.

_Also change reamde_

Reply:
done